dev/com.ibm.ws.microprofile.reactive.messaging_fat/fat/src/com/ibm/ws/microprofile/reactive/messaging/fat/kafka/invalid/badconfig/KafkaBadConfigTest.java

```diff
@@ -177,7 +194,12 @@ public void testBadConfigOutgoing() throws Exception {
 
         // Check that the bad config error was emitted
         List<String> configErrorLines = server.findStringsInLogsUsingMark("CWMRX1008E:", server.getDefaultLogFile());
-        assertThat(configErrorLines, hasSize(1));
+        //Due to logging changes older versions of OL emit the expected code less frequently
```

Comment:
need to change comment to be about older feature sets not OL itself

Reply:
Which bit of code did we change that makes these messages get logged more times?